.github/workflows/windows.yml

```diff
@@ -2,74 +2,115 @@ name: Windows
 
 on:
   push:
-    branches: [ dev ]
+    branches: [ dev, ci ]
```

Comment:
Is `ci` branch temporary?

Reply:
Yes, it's my forked repo branch to test the changes. I needed this since I couldn't get the forked repo to build my changes because of self-hosted setting.